docker-compose.yml

```diff
@@ -34,27 +36,6 @@ services:
     - mailcatcher
     - cache
 
-  rails-test:
```

Comment:
The README still references the test container, I guess now you just use `rails`?
https://github.com/hitobito/development/blob/e52d17050a322b8fec7d0f3e50ce8e7d6f1297aa/README.md#L126